src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -157,6 +158,30 @@ public function getEnv($prefix, $name, \Closure $getEnv)
             return $env;
         }
 
+        if ('url' === $prefix) {
+            $env = \parse_url($env);
+
+            if ($env === false) {
+                throw new RuntimeException(sprintf('Invalid URL in env var "%s"', $name));
+            }
+
+            $env += array(
+                'scheme' => '',
+                'host' => '',
+                'port' => 0,
+                'user' => '',
+                'pass' => '',
+                'path' => '',
+                'query' => '',
+                'fragment' => ''
+            );
+
+            # remove the '/' separator
+            $env['path'] = ($env['path'][0] ?? '') === '/' ? substr($env['path'], 1) : $env['path'];
```

Comment:
Why not: `ltrim($env['path'], '/')`?

Reply:
I think you missed something how `ltrim()` works, check: https://3v4l.org/GfkPX

Reply:
```
http://localhost//path
                ^^
                |`---- part of the path
                `----- separator
```
Splitting the url should return
```yaml
host: 'localhost'
path: '/path'
```